README.md

```diff
@@ -13,16 +13,22 @@ layout:
+# Kontrol
+
+Struggling to kontrol your smart contracts behave as intended, even after thorough unit testing, fuzzing, and other validation procedures? Frustrated with finding bugs in your code but unable to prove their absence?
+
+[**Kontrol**](https://github.com/runtimeverification/kontrol) is designed for you!
+
 
-[**KEVM**](https://github.com/runtimeverification/evm-semantics) is a tool that enables formal verification of smart contracts on the [Ethereum](https://ethereum.org/en/) blockchain. It provides a mathematical foundation for specifying and implementing smart contracts. Developers can use **KEVM** to rigorously reason about the behavior of their smart contracts, ensuring correctness and reducing the likelihood of vulnerabilities in the contract code.
+- **Open Source**: Kontrol is open source under the [BSD-3 Clause License](https://github.com/runtimeverification/kontrol/blob/master/LICENSE). This gives you full Kontrol to use, modify, and redistribute it according to your needs.
+- **Easy to Use**: Take advantage of the languages and tests you’re already familiar with. Kontrol enables developers to write formal specifications as [Foundry](https://book.getfoundry.sh/) test suites in [Solidity](https://soliditylang.org). It allows developers to reuse their existing test suites for formal verification. Even if you're not a verification engineer, Kontrol lets you formally verify your contracts in minutes.
+- **Scalable**: Kontrol automatically performs theorem proving on your specifications against your contracts through composable symbolic execution. It allows you to manipulate the proof process and provide simplification rules to achieve any proof goal. This ensures you can verify your contracts at any scale.
+- **Trustworthy**: Kontrol has a trustworthy mathematical foundation for specifying and verifying smart contracts. It is built on the open-source, validated, and intuitive formal semantics of Ethereum Bytecode, [KEVM](https://github.com/runtimeverification/evm-semantics). Thus, Kontrol how your smart contracts execute on the virtual machine exactly as you formally verified them.
 
-[Foundry](https://book.getfoundry.sh/) is a smart contract development toolchain. It manages dependencies, compiles projects, runs tests, facilitates deployments and provides a command-line interface to interact with the chain via [Solidity](https://soliditylang.org/) scripts. If you’re curious about Foundry, have a look at one of our blog posts about it [here](https://runtimeverification.com/blog/foundry-gen-2-of-ethereum-tooling).
+Even better, [KaaS](https://docs.runtimeverification.com/kaas) is a cloud-based solution that allows you to perform CI-Integrated formal verification on your contracts with speed and ease.
```

Comment:
- **CI Integrated**: It addition to running locally, Kontrol can also be integrated into your project's CI for continuous assurance and perform cloud-based verification via [KaaS](https://docs.runtimeverification.com/kaas).

Reply:
I've added this information to the first paragraph. What do you think?

Reply:
I think what Palina is saying in her comment is that we should add a "CI Integrated:..." Bullet to the list above this paragraph so that it is more consistent with this section. I personally think it looks better that way, but I don't think it looks or reads poorly in either case.